app/routes/circulars._archive._index/route.tsx

```diff
@@ -373,7 +373,7 @@ export default function () {
         <>
           {isGroupView ? (
             <SynonymGroupIndex
-              allItems={items as SynonymGroupWithMembers[]}
+              allItems={items as SynonymGroup[]}
```

Comment:
Why is a type assertion necessary here?

Reply:
<img width="947" alt="Screenshot 2025-05-09 at 12 55 59 PM" src="https://github.com/user-attachments/assets/892732de-ec6e-4001-868a-a1683088ee26" />

Reply:
I see.